ssn/integrated/informativeExamples/InkBird-IBS-TH2-instance.ttl

```diff
@@ -0,0 +1,224 @@
+    sosa:hasFeatureOfInterest <00001/acmePorterSixPack> ;
+    sosa:resultTime "2024-02-20T01:35:45Z"^^xsd:dateTime ;
+    sosa:hasSimpleResult "12"^^unit:DEG_C .
+
+<observation1b> rdf:type sosa:Observation ;
+    rel:own <acmeBreweryCo> ;
+    sosa:observedProperty <airRelativeHumidityOfProduct> ;
+    sosa:madeBySensor <12345/HumiditySensor>  ;
+    sosa:hasFeatureOfInterest <00001/acmePorterSixPack> ;
+    sosa:resultTime "2024-02-20T01:35:45Z"^^xsd:dateTime ;
+    sosa:hasSimpleResult "60"^^unit:PERCENT .
+
+# The battery is a special case because it's property is linked to the battery, not to the environment.
+
+<observation1c> rdf:type sosa:Observation ;
+    rel:own <acmeBreweryCo> ;
+    sosa:observedProperty  <2024/02/batteryPack>;
+    sosa:madeBySensor <12345/BatterySensor>  ;
+    sosa:hasFeatureOfInterest <12345/BatterysHealth> ;
+    sosa:resultTime "2024-02-20T01:35:45Z"^^xsd:dateTime ;
+    sosa:hasSimpleResult "99"^^unit:PERCENT .
+
+<breweryObserver> a prov:Activity ;
+    rdfs:comment "Brewery operating system logging process." ;
+    prov:atLocation <acmeBreweryCo> ;
+    prov:generated <observation1a> ;
+    prov:generated <observation1b> ;
+    prov:generated <observation1c> ;
+    prov:wasStartedBy <alice> . # She turned it on last time.     
+
+#
+# Product is loaded on the truck
+#
+
+    
+<observation67a> a sosa:Observation, prov:Entity ;
+    rel:own <acmeBreweryCo> ;
+    rel:own <acmeDelivery> ;
+    sosa:observedProperty <airTemperatureOfProduct> ;
+    sosa:madeBySensor <12345/TemperatureSensor>  ;
+    sosa:hasFeatureOfInterest <00001/acmePorterSixPack> ;
+    sosa:resultTime "2024-02-22T04:15:05Z"^^xsd:dateTime ;
+    sosa:hasSimpleResult "19"^^unit:DEG_C .
+
+<observation67b> rdf:type sosa:Observation,prov:Entity ;
+    rel:own <acmeBreweryCo> ;
+    rel:own <acmeDelivery> ;
+    sosa:observedProperty <airRelativeHumidityOfProduct> ;
+    sosa:madeBySensor <12345/HumiditySensor>  ;
+    sosa:hasFeatureOfInterest <00001/acmePorterSixPack> ;
+    sosa:resultTime "2024-02-22T04:15:05Z"^^xsd:dateTime ;
+    sosa:hasSimpleResult "74"^^unit:PERCENT .
+
+<observation67c> rdf:type sosa:Observation,prov:Entity ;
+    rel:own <acmeBreweryCo> ;
+    rel:own <acmeDelivery> ;
+    sosa:observedProperty  <2024/02/batteryPack>;
+    sosa:madeBySensor <12345/BatterySensor>  ;
+    sosa:hasFeatureOfInterest <12345/BatterysHealth> ;
+    sosa:resultTime "2024-02-22T04:15:05Z"^^xsd:dateTime ;
+    sosa:hasSimpleResult "99"^^unit:PERCENT .
+
+<TruckObserver> a prov:Activity ;
+    rdfs:comment "Truck onboard monitoring system" ;
+    prov:atLocation <acmeTruck> ;
+    prov:generated <observation67a> ;
+    prov:generated <observation67b> ;
+    prov:generated <observation67c> ;
+    prov:startedAtTime "2024-02-22T04:15:05Z"^^xsd:dateTime ;
+    prov:endedAtTime "2024-02-22T05:55:38Z"^^xsd:dateTime ;
+    prov:wasStartedBy <driver> ;
+    prov:atLocation <parkingLot> .
+
+<parkingLot> a prov:Location, geosparql:Geometry; 
```

Comment:
A parking lot would rather be an instance of geosparql:Feature (or better one of its Point sub-class), with an associated instance of geosparql:Geometry.